tests/scripts/build-release.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -ex
+
+#############
+# FUNCTIONS #
+#############
+
+function  build() {
+    # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+    build/run make VERSION=0 build.all
+    # quick check that go modules are tidied
+    build/run make mod.check
+}
+
+if echo "${BRANCH_NAME}" | grep "release-" ; then
+    build
+    build/run make -C build/release build BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=true GIT_API_TOKEN=${GIT_PSW}
+    git status &
+    git diff &
+    build/run make -C build/release publish BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=true AWS_ACCESS_KEY_ID=${AWS_USR} AWS_SECRET_ACCESS_KEY=${AWS_PSW} GIT_API_TOKEN=${GIT_PSW}
+elif echo "${BRANCH_NAME}" | grep "master\|tag" ; then
```

Comment:
```suggestion
elif echo "${BRANCH_NAME}" | grep -E "master|tag" ; then
```

Are you trying to match one of two?

Reply:
Yes...

Reply:
Actually go with:

```sh
elif [[ ${BRANCH_NAME} =~ master|tag ]]; then
```

Reply:
done.